Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:latest
```

Comment:
I'd presonally use `debian:stable-slim`. Any reason as to why we use ubuntu in this case?

Reply:
nope. Will switch to `debian:stable-slim`.